Cesium3DTilesContent/test/TestQuantizedMeshLoader.cpp

```diff
@@ -678,6 +678,47 @@ static void checkGeneratedGridNormal(
   }
 }
 
+static void checkGltfSanity(const Model& model) {
+  CHECK(model.asset.version == "2.0");
+  REQUIRE(model.scene >= 0);
+  REQUIRE(size_t(model.scene) < model.scenes.size());
+  CHECK(!model.getSafe(model.scenes, model.scene).nodes.empty());
+
+  for (const Buffer& buffer : model.buffers) {
+    CHECK(buffer.byteLength > 0);
```

Comment:
I'd like to add one more check for sanity:

```suggestion
    CHECK(buffer.byteLength > 0);
    CHECK(buffer.byteLength == static_cast<int64_t>(buffer.cesium.data.size()));
```